client/guidestours/config.js

```diff
@@ -1,26 +1,59 @@
+const React = require( 'react' );
+const Gridicon = require( 'components/gridicon' );
+const i18n = require( 'lib/mixins/i18n' );
+
 const config = {
 	init: {
-		text: 'Welcome to WordPress.com!',
+		text: i18n.translate( '{{strong}}Welcome to WordPress.com!{{/strong}} {{gridicon/}} We\'ll show you around a bit now.', {
+			components: {
+				strong: React.createElement( 'strong' ),
```

Comment:
It does work, you're right. It didn't work previously, but that was probably because I had overlooked something else.